content/en-us/reference/engine/classes/Debris.yaml

```diff
@@ -3,64 +3,61 @@ type: class
 category: Utility
 memory_category: Instances
 summary: |
-  The Debris service allows the developer to schedule the removal of the object
-  without yielding any code, through the usage of the `Class.Debris:AddItem()`
+  The Debris service allows the developer to schedule guaranteed destruction of
+  the object without yielding, through the usage of the `Class.Debris:AddItem()`
   method.
 description: |
-  The Debris service allows the developer to schedule the removal of the object
-  without yielding any code, through the usage of the `Class.Debris:AddItem()`
+  The **Debris** service allows scheduling of guaranteed destruction of an
+  object without yielding, through the usage of the `Class.Debris:AddItem()`
   method.
 
-  After the lifetime argument has elapsed (in seconds) the object is removed in
-  the same manner as `Class.Instance:Destroy()`.
-
   As Debris is a service it must be created using the
   `Class.ServiceProvider:GetService()` method.
 
-  **Why use Debris?**
+  #### Advantages
```

Comment:
Mostly because I dislike the tone of "asking questions" in technical docs. :)

Reply:
Just kept the header from before, but yeah thanks for keeping the docs consistent and neutral